doc/src/gotchas.rst

```diff
@@ -758,6 +758,104 @@ that you want, and they will all be evaluated according to the function.
 
     See the Python docs for more information on function parameters.
 
+
+Classification of the objects
+=============================
```

Comment:
This should go in its own documentation page. It isn't really a gotcha. Furthermore, I think the kind system won't be relevant to most end-users of SymPy.

Reply:
Where should the document page go? Perhaps in `special topics`?

Reply:
I moved it to `special topics` because there is nowhere else to put it.